README.md

```diff
@@ -190,6 +201,8 @@ To use an encoder, specify the name, and the type "encoder".
 
 Evasions are the modules that help you to evade AV/EDR:
 
+#### Golang
```

Comment:
any chance we can collapse all of the same injectors/evasions/encoders into the same table.. for example: looks cleaner if we have one table with all the encodings for c and go rather than two separate

Reply:
The problem i think is that in the future maybe be for some artifacts we will have additionnal comment for one language but not the others

I was also thinking to move the doc to something more complete to the readthedoc platform (https://merlin-c2.readthedocs.io/en/latest/ for example)... but as the tool is use by a few people i dont know if it would be very interesting